[JBTM-3477] Change the semantics of the annotation attribute

marcosgopen opened the pull request:
Change the semantics of the annotation attribute SRA#delayCommit

[https://issues.redhat.com/browse/JBTM-3477](https://issues.redhat.com/browse/JBTM-3477) 


CORE !TOMCAT !AS_TESTS RTS !JACOCO !XTS !QA_JTA !QA_JTS_JACORB !QA_JTS_JDKORB !QA_JTS_OPENJDKORB !PERF LRA !DB_TESTS !mysql !db2 !postgres !oracle

jmfinelli commented:
Thank you very much for your PR @marcosgopen! I personally welcome PRs that reformat the code of java files (and others), but I think you should use two commits to do so:
* First commit to reformat the file you are modifying
* Second commit to carry out the required modifications (e.g. to address a JIRA or other issues)
So, could you please restructure your PR?

marcosgopen commented:
Thank you @jmfinelli, I have splitted the PR in 3 commits as you suggested.

CORE !TOMCAT !AS_TESTS !RTS !JACOCO !XTS !QA_JTA !QA_JTS_JACORB !QA_JTS_JDKORB !QA_JTS_OPENJDKORB !PERF !LRA !DB_TESTS !mysql !db2 !postgres !oracle JDK11

mmusgrov commented:
> Thank you @jmfinelli, I have splitted the PR in 3 commits as you suggested.
> 
> CORE !TOMCAT !AS_TESTS !RTS !JACOCO !XTS !QA_JTA !QA_JTS_JACORB !QA_JTS_JDKORB !QA_JTS_OPENJDKORB !PERF !LRA !DB_TESTS !mysql !db2 !postgres !oracle JDK11

FYI, the pull job script (scripts/hudson/narayana.sh) only reads the pull body (`curl https://api.github.com/repos/jbosstm/narayana/pulls/1935`) so this will be ignored. If you want to alter any of the axes used to run the job then edit the initial PR comment.

marcosgopen commented:
Thanks @mmusgrov for the requests. I added an afterClass method for closing not ended SRA transactions. 
Thanks @jmfinelli for your suggestions, I squashed refactoring commits into one and renamed it.

mmusgrov reviewed (approved):
It looks great, thanks for the useful contribution.
I think you should have merge rights, though I think we still need approval from @jmfinelli

jmfinelli commented:
Thank you very much @marcosgopen for your contribution :smile: :tada:
